testing/fetcher.ts

```diff
@@ -703,6 +704,15 @@ class AuthenticatingBlobStorage implements TemporaryBlobStorage {
   }
 }
 
+class AuthenticatingSyncStateService implements SyncStateService {
```

Comment:
Hmm I don't know why I choose to call the existing thing AuthenticatingBlobStorage when it's fake. This should probably be `FakeSyncStateService`.

Reply:
Sounds good